ts/DataGrid/DataGrid2/DataGridColumn.ts

```diff
@@ -0,0 +1,171 @@
+/* *
+ *
+ *  Data Grid class
+ *
+ *  (c) 2020-2024 Highsoft AS
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ *  Authors:
+ *  - Dawid Dragula
+ *
+ * */
+
+'use strict';
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import DataGridCell from './DataGridCell.js';
+import DataGridTable from './DataGridTable.js';
+import DataTable from '../../Data/DataTable.js';
+import Globals from './Globals.js';
+
+
+/* *
+ *
+ *  Class
+ *
+ * */
+
+/**
+ * Represents a column in the data grid.
+ */
+class DataGridColumn {
+
+    /* *
+    *
+    *  Properties
+    *
+    * */
+
+    /**
+     * The viewport (table) the column belongs to.
+     */
+    public viewport: DataGridTable;
+
+    /**
+     * The width ratio of the column.
```

Comment:
Could you please explain a bit more what it does and why?
As far as I see there is no way to set it right?